extensions/interactions/GraphInput/directives/GraphInput.js

```diff
@@ -429,8 +434,48 @@ oppia.directive('graphViz', [
                 $scope.canEditVertexLabel) {
               beginEditVertexLabel(index);
             }
+            if ($scope.isMobile) {
+              if ($scope.state.addEdgeVertex === null &&
+                  $scope.state.currentlyDraggedVertex === null) {
+                $scope.onTouchInitialVertex(index);
```

Comment:
Ditto here and elsewhere, if the method isn't used in the template, I don't think you need to attach it to $scope.